src/js/tabs/onboard_logging.js

```diff
@@ -338,6 +338,16 @@ onboard_logging.initialize = function (callback) {
                 {text: "GPS_DOP"},
                 {text: "FAILSAFE"},
                 {text: "GYRO_CALIBRATION"},
+                {text: "ANGLE_MODE"},
+                {text: "ANGLE_TARGET"},
+                {text: "CURRENT_ANGLE"},
```

Comment:
These 3 are in master now.

Reply:
@damosvil , merge master into branch or rebase branch upon master should fix it.  i recall betaflight team prefers rebase, if i'm not mistaken.

Reply:
rebased to current master